### DISCUSSION TITLE
ver 8.5.7 is not working in my Dreambox 900 (fixed)

### DISCUSSION THREAD

**Discussion by ahd299** [OpenCockpit/TMDBCockpit#5] [General]
Hello,

after the upgrade to version 8.5.7, the TMDBCockpit pluging not longer shows in my dreambox 900 plugin list (i got a message some plugins are not available: Extensions/TMDBCockpit ('en_EN')...it seems there is an error preventing it from start.

Can you please fix i?

Thanks,

**Comment by xcentaurix** [comment]
sorry for that... which language are you using?

**Reply by ahd299** [reply]
English

**Reply by xcentaurix** [reply]
that's what i thought ;-)
please try 8.5.8 ... it should fix the bug. thanks.

**Reply by ahd299** [reply]
thanks for the follow-up, the same problem persist also in ver 8.5.8 :(

**Reply by xcentaurix** [reply]
hmmm, can you do me a favor and delete the line "config.plugins.tmdb.lang=en_EN" in /etc/enigma2/settings ?
first stop enigma2: systemctl stop enigma2 then start it again: systemctl start enigma2

**Reply by ahd299** [reply]
thanks for ver. 8.5.9, the problem has been fixed for good

**Comment by xcentaurix** [comment]
well not really... i just reverted the change... it's now equivalent to 8.5.6 again.... need to think about it a bit longer.

**Reply by ahd299** [reply]
good luck and thanks for your prompt support and enjoy the weekend

**Reply by ahd299** [reply]
ver. 8.5.10 is good with english, I hope you figured the problem and it's surely fixed :)

**Comment by xcentaurix** [comment]
thanks for the feedback. yes, I figured it out and it is fixed. the bug just affected the english language... en_EN vs. en_GB :-)
